src/LaunchDarkly/GuzzleFeatureRequester.php

```diff
@@ -29,7 +29,7 @@ public function __construct($baseUri, $sdkKey, $options)
         $this->_logger = $options['logger'];
         $stack = HandlerStack::create();
         if (class_exists('Kevinrob\GuzzleCache\CacheMiddleware')) {
-            $stack->push(new CacheMiddleware(new PublicCacheStrategy(isset($options['cache']) ? $options['cache'] : null), 'cache'));
+            $stack->push(new CacheMiddleware(new PublicCacheStrategy(isset($options['cache']) ? $options['cache'] : null)), 'cache');
```

Comment:
The `'cache'` value should probably be an argument to [`HandlerStack::push($middleware, $name)`](https://github.com/guzzle/guzzle/blob/6.2.1/src/HandlerStack.php#L139-L143) instead of to the constructor of [`CacheMiddleware`](https://github.com/Kevinrob/guzzle-cache-middleware/blob/v1.4.1/src/CacheMiddleware.php#L53-L58), which only accepts a single argument. 

That is, as of the moment, the `'cache'` value is currently not used.